language/Turkish/strings.po

```diff
@@ -1739,31 +1739,31 @@ msgstr ""
 
 #: src/frontend/mame/ui/selmenu.cpp:642
 msgid "Graphics: Unimplemented, "
-msgstr ""
+msgstr "Grafik: Uygulanmamış"
 
 #: src/frontend/mame/ui/selmenu.cpp:644
 msgid "Graphics: Imperfect, "
-msgstr ""
+msgstr "Grafik: Kusurlu"
 
 #: src/frontend/mame/ui/selmenu.cpp:646
 msgid "Graphics: OK, "
-msgstr ""
+msgstr "Grafik: Tamam"
 
 #: src/frontend/mame/ui/selmenu.cpp:649
 msgid "Sound: None"
-msgstr ""
+msgstr "Ses: Yok"
```

Comment:
Here "Sound: None" is translated as "Ses: Yok", however previously "None" is translated as "Hiçbiri".  In both cases it's used in the same context (one is used in the default UI, the other with `-ui simple`).  Is there a reason the translations need to be different?

Reply:
i guess this is used for info and not for choice and means that driver/machine/system has no sound.
"Hiçbiri" is used when choosing something from multiple things and represent 'none of them'.
i hope i understood right and i told it right